src/components/Card.js

```diff
@@ -89,13 +89,21 @@ export const Card = () => {
               value={network.ssid}
               onChange={(e) => setNetwork({ ...network, ssid: e.target.value })}
             />
-            <label className={network.hidePassword ? 'no-print' : ''}>
+            <label
+              className={[
```

Comment:
Rather than using an array and joining back to a string, you can use template literals:

```js
className={`
  ${network.hidePassword ? 'no-print hidden' : ''}
  ${network.encryptionMode === 'nopass' ? 'hidden' : ''}
`}
```

Then maybe instead of using a ternary you could and it, since we don't need the falsey value:
```
className={`
  ${network.hidePassword && 'no-print hidden'}
  ${network.encryptionMode === 'nopass' && 'hidden'}
`}
```

This might be a cleaner pattern to use for these, what do you think?

Reply:
Yeah that definitely looks cleaner! Making those changes now